configure.ac

```diff
@@ -154,7 +154,7 @@ AC_ARG_ENABLE([fuzzer],
   [
 case "$enableval" in
     no) fuzzer= ;;
-    yes) fuzzer="-fsanitize=fuzzer" ;;
+    yes) fuzzer="$LIB_FUZZING_ENGINE" ;;
```

Comment:
You might wonder why use `$LIB_FUZZING_ENGINE` instead of `-fsanitize=fuzzer`.

- Fuzzing is not synonymous with libFuzzer.
- Using this approach makes it easier to utilize different fuzzers with the same code. For example: `centipede`, `honggfuzz`, `AFL++`, and libFuzzer.

Reply:
In this case, add alternate values to yes/no (like this is done for sanitizers.

Reply:
Oh, alright.